Add abandoned log status (#945)
farmOS/farmOS: docs/model/type/log.md

#### Status

Logs can be marked as "pending", "done", or "abandoned". Pending Logs represent
planned events, while done Logs represent events that actually happened. If an

> event is never completed, and there is no intention of completing it, the :og

Reviewer: ```suggestion
event is never completed, and there is no intention of completing it, the log
```

Author: Oops good catch @paul121. Force pushed this fix. :-)

> event is never completed, and there is no intention of completing it, the Log

Reviewer 2: Nit: unnecessary comma

```suggestion
event is never completed and there is no intention of completing it, the Log
```

Author: Good eye! Fixed!

> can be marked as "abandoned" to preserve it for future reference.

Reviewer 2: Nit:

```suggestion
can be marked as "abandoned" to make that clear while preserving it for future reference.
```

Author: :+1: Done!
